newIDE/app/src/InAppTutorial/HTMLUtils.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+export const AboveMaterialUIMaxZIndex = 1501; // highest z-index used by MaterialUI is 1500
```

Comment:
ubernit: this is not a class/component, so camelCase:
```suggestion
export const aboveMaterialUIMaxZIndex = 1501; // highest z-index used by MaterialUI is 1500
```